javascript/main.js

```diff
@@ -9,52 +23,88 @@ var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
 
 
-function printTime() {
+// Each second we need to update our screen. So go ahead 
+// and create a function that will receive the value for minutes
+// and seconds, and print that on our HTML.
+// Use our Chronometer methods to get the values.
 
+function printTime(minutes, seconds) {
+    printMinutes(minutes);
+    printSeconds(seconds);
+    //printMilliseconds(milliseconds);
 }
 
-function printMinutes() {
-
+function printMinutes(minutes) {
+    minDec.innerHTML = minutes[0];
+    minUni.innerHTML = minutes[1];
 }
 
-function printSeconds() {
-
+function printSeconds(seconds) {
+    secDec.innerHTML = seconds[0];
+    secUni.innerHTML = seconds[1];
 }
 
-function printMilliseconds() {
+// function printMilliseconds(milliseconds) {
+//     milDec.innerHTML = milliseconds[0];
+//     milUni.innerHTML = milliseconds[1];
+// }
 
-}
 
 function printSplit() {
-
+    var splitList = document.querySelector("#splits");
+    var newSplit = `${minDec.innerHTML} ${minUni.innerHTML} : ${secDec.innerHTML} ${secUni.innerHTML}`;
+    var li = document.createElement('li');
+    li.innerHTML = newSplit;
+    splitList.appendChild(li);
 }
 
 function clearSplits() {
-
+    minDec.innerHTML = 0;
+    minUni.innerHTML = 0;
+    secDec.innerHTML = 0;
+    secUni.innerHTML = 0;
```

Comment:
here you could also have linked this function to the chronometer resetClick() function (where the logic happens), and where you could reset the currentTime to 0 and setTime() again:

Chronometer.prototype.resetClick = function () {
  this.currentTime = 0;
  this.setTime();
  clearSplits();
};

function clearSplits() {
    var splitList = document.querySelector("#splits");
    splitList.innerHTML = '';
}